apps/src/jigsaw/jigsaw.js

```diff
@@ -181,22 +180,12 @@ Jigsaw.init = function (config) {
     Jigsaw.successListener = Blockly.mainBlockSpaceEditor.addChangeListener(
       function (evt) {
         // Only used by level1, in which the success criteria is clicking on the block
-        // Only effective with Google Blockly.
-        if (evt.type === 'click' && evt.blockId === 'block1') {
+        if (evt.type === 'click' && evt.blockId === 'jigsaw_1A') {
           Jigsaw.block1Clicked = true;
         }
         checkForSuccess();
       }
     );
-
-    // Only used by level1, in which the success criteria is clicking on the block
```

Comment:
Not quite following- How can this be fully deleted and not replaced?

Reply:
The APIs between CDO Blockly and Google Blockly are different. In the case of this functionality, there were two separate checks for this success criteria, one for each Blockly version. Now that we are migrating the lab to Google Blockly, we will rely upon the change listener that is add in the lines directly above this.